pkgs/desktops/mate/default.nix

```diff
@@ -10,7 +10,7 @@ let
       pkgs.genericUpdater {
         inherit pname version odd-unstable;
         attrPath = "mate.${pname}";
-        versionLister = "${pkgs.common-updater-scripts}/bin/list-archive-two-level-versions ${url}";
```

Comment:
I wonder if we should have a function that creates this text `pkgs.common-updater-scripts.versionListers.archiveTwoLevelVersions { inherit url; }`.

Reply:
Instead I am going to add the function `httpTwoLevelsUpdater`, which calls `genericUpdater` with this argument.

Reply:
I guess that solves the issue of having to specify the command manually but it is even less compositional than the original variant. I remember wanting to use `gitUpdater` for the community part of gmic-qt but it just could not mesh with the rest:

https://github.com/NixOS/nixpkgs/blob/1822c35423bb208cbdcf6ad75058b38cd5cff88e/pkgs/tools/graphics/gmic-qt/default.nix#L148

Reply:
The more general function `genericUpdater` is still available.